neuralprophet/df_utils.py

```diff
@@ -1236,6 +1236,29 @@ def get_dist_considering_two_freqs(dist):
     return f1 + f2
 
 
+def _get_dominant_frequency_percentage(frequencies, distribution, filter_list) -> float:
+    """Calculate dominant frequency percentage of dataframe.
+
+    Parameters
+    ----------
+        frequencies : list
+            list of numeric delta values (``ms``) of frequencies
+        distribution : list
+            list of occasions of frequencies
+        filter_list : list
+            list of frequencies to be filtered
+
+    Returns
+    -------
+        float
+            Percentage of dominant frequency within the whole dataframe
+
+    """
+    dominant_frequencies = [freq for freq in frequencies if freq in filter_list]
+    dominant_distribution = [distribution[np.where(frequencies == freq)] for freq in dominant_frequencies]
+    return sum(dominant_distribution) / sum(distribution)
```

Comment:
@noxan Can we ensure that we never divide by zero? I am not too deep into this function, but thought we should think abou it.

Reply:
this would mean the dataset is empty